lib/modules/manager/sbt/extract.ts

```diff
@@ -289,13 +323,21 @@ const query = q.tree<Ctx>({
   postHandler: registryUrlHandler,
 });
 
+// Extract 1 file
 export function extractPackageFile(
```

Comment:
`extractPackageFile` is reserved for the upper-level API usage, please rename it to avoid confusion

Reply:
renamed